Makefile

```diff
@@ -26,8 +30,10 @@ install-tools:
 	go get -u github.com/modocache/gover/...
 	go get -u github.com/mattn/goveralls/...
 lint: install
-	scripts/lint.sh
+	$(ROOT_DIR)/scripts/lint.sh
 test:
-	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
+	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s ./...
+test-full:
+	$(ROOT_DIR)/scripts/test-full.sh
 test-verbose:
```

Comment:
Let's discuss that in the other PR https://github.com/go-clang/bootstrap/pull/11#r79457739